packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -85,10 +96,12 @@ export interface FileSharingOptions {
    * @beta
    */
   multiple?: boolean;
+  /* @conditional-compile-remove-from(stable): FILE_SHARING */
   /**
    * A function of type {@link FileUploadHandler} for handling file uploads.
    * @beta
    */
+  /* @conditional-compile-remove-from(stable): FILE_SHARING */
```

Comment:
Not needed, you already have one on line 99.

Reply:
In the comments the link is mentioned `{@link FileUploadHandler}` and `FileUploadHandler`  is conditionally compiled so it throws error in build